Dockerfile

```diff
@@ -0,0 +1,54 @@
+
+# s2i-haproxy
+FROM openshift/base-centos7
+
+ENV HAPROXY_DOWNLOAD "https://www.haproxy.org/download/1.8/src/haproxy-1.8.0.tar.gz"
+ENV HAPROXY_MD5 6ccea4619b7183fbcc8c98bae1f9823d
```

Comment:
Why ENV and not ARG ? It could ask for HAPROXY_VERSION and HAPROXY_MD5 by ARG (and default to these). Then you could embed the HAPROXY_VERSION in the display-name/description.

Reply:
Good point! fixing

Reply:
The same for LUA_*.